hedera-mirror-rest/controllers/contractController.js

```diff
@@ -748,8 +744,20 @@ class ContractController extends BaseController {
     if (rows.length !== 1) {
       throw new NotFoundError();
     }
-
-    res.locals[responseDataLabel] = formatContractRow(rows[0], ContractBytecodeViewModel);
+    const contract = rows[0];
+    if(contract.file_id !== null) {
+      const {query, params} = FileDataService.getFileData(contract.file_id, contract.created_timestamp);
+      if (logger.isTraceEnabled()) {
+        logger.trace(`getFileData query: ${query}, params: ${params}`);
+      }
+      const {rows} = await pool.queryQuietly(query, params);
+      if (rows.length === 1) {
+        contract.bytecode = rows[0].data;
+      }
+    } else {
+      contract.bytecode = contract.initcode.toString('hex');
```

Comment:
`file_id` and `initcode` can both be null. we also need a integration test

```suggestion
      contract.bytecode = contract.initcode?.toString('hex');
```

Reply:
Please let the reviewer resolve their own conversations. As Xin notes not all items were addressed so it's important they double check things before resolving.

Reply:
I did not resolve it. I did commit the suggestion which auto resolved it. And I am adding an integration test now.

Reply:
learned a new trick 🤣